### ISSUE TITLE
Allow container reordering

### ISSUE THREAD

**Issue by maciekmm** [maciekmm/container-tabs-sidebar#7]

**Comment by LeeteqXV** [comment]
Now that the following has been fixed in the new Firefox v.123;
https://bugzilla.mozilla.org/show_bug.cgi?id=1333395
(Container tabs sort order), 
- it seems that this addon can now be updated to allow us to re-order the containers?
- if I am not mistaken, that "fix" in Firefox is not actually putting a reordering feature in the GUI, just making it possible for addons to do that? I can find no visible sign of that change in Firefox.
This is the missing part here, really need to be able to (re-)order the containers.

**Comment by maciekmm** [comment]
That's amazing. Thanks for letting me know.

I'll see what I can come up with :)

**Comment by maciekmm** [comment]
@LeeteqXV
You can find the initial implementation here: https://github.com/maciekmm/container-tabs-sidebar/tree/feature/container-reordering
I need to clean it up and do some more tests, especially with temporary containers and some auto-hiding ones, but if you can please provide any feedback.

**Comment by LeeteqXV** [comment]
Great job :-) **Tested: Seems to work in a stable manner.**

FTR: I created an empty, new profile with the --profileManager, then copied all the files from an existing, relatively new Firefox profile (that was created with FF-v123) into it. There, I had already a Container Tabs setup. I disabled the existing Container tabs addon, opened a tab to about:debugging#/runtime/this-firefox , and added the test-version by opening the unpacked zip file contents at (Downloads)/container-tabs-sidebar-feature-container-reordering/src/manifest.json from the Load Temporary Addon button. (Detailing just in case some newbies want an easy start)

I first went to the Firefox settings page, clicked on the Container Tabs Settings button, and on that page I was hoping to find some icons along the left side to drag/reorder the tabs. But that page might be more of a challenge to customize? Nothing there, so I guessed that I could drag and drop the containers directly in the sidebar, which works like a charm.

I have worked a little with that firefox and it seems to not break anything.
I closed the firefox, re-opened it, and that temporary change was not visible because then that addon was unloaded (hence the wording on the button; "Load Temporary Addons").
I went back to that page, and loaded it again, and lo-and-behold, the changes after the manual reordering of the containers are kept/still there/not lost. :-)

**Suggestion:**
I suggest to clearly mention this possibility (drag-n-drop directly in the sidebar) should be mentiond on top of the addon Preferences page, so everyone will be aware of it.
